co-authors-plus.php

```diff
@@ -380,14 +380,18 @@ public function coauthors_meta_box( $post ) {
 				<?php
 				foreach ( $coauthors as $coauthor ) :
 					$count++;
+					$avatar_url = '';
+					if ( has_post_thumbnail( $coauthor->ID ) ) {
+						$avatar_url = get_the_post_thumbnail_url( $coauthor->ID, $this->gravatar_size );
+					}
 					?>
 					<li>
-						<?php echo get_avatar( $coauthor->user_email, $this->gravatar_size ); ?>
 						<span id="<?php echo esc_attr( 'coauthor-readonly-' . $count ); ?>" class="coauthor-tag">
 							<input type="text" name="coauthorsinput[]" readonly="readonly" value="<?php echo esc_attr( $coauthor->display_name ); ?>" />
 							<input type="text" name="coauthors[]" value="<?php echo esc_attr( $coauthor->user_login ); ?>" />
 							<input type="text" name="coauthorsemails[]" value="<?php echo esc_attr( $coauthor->user_email ); ?>" />
 							<input type="text" name="coauthorsnicenames[]" value="<?php echo esc_attr( $coauthor->user_nicename ); ?>" />
+							<input type="text" name="coauthorsavaters[]" value="<?php echo esc_attr( $avatar_url ); ?>" />
```

Comment:
Not a big deal, but can we fix the typo here of `avatar`? It should be `coauthorsavatars`.